app/assets/stylesheets/active_admin/_header.scss

```diff
@@ -1,58 +1,48 @@
 //= depend_on_asset "active_admin/nested_menu_arrow.gif"
 //= depend_on_asset "active_admin/nested_menu_arrow_dark.gif"
 // ----------------------------------- Header
-#header {
-  @include primary-gradient;
-  @include shadow;
-  @include text-shadow(#000);
+.header {
+  background-color: lighten($primary-color, 5%);
   display: table;
-  height: 20px;
   width: 100%;
-  overflow: visible;
-  position: inherit;
   padding: 5px 0;
   z-index: 900;
 
+  a, a:link, a:visited {
+    color: $page-header-text-color;
+    text-decoration: none;
+  }
+
+  a:hover {
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.